src/controllers/controllerengine.cpp

```diff
@@ -76,6 +76,8 @@ Output:  -
 -------- ------------------------------------------------------ */
 void ControllerEngine::callFunctionOnObjects(QList<QString> scriptFunctionPrefixes,
                                              const QString& function, QScriptValueList args) {
+    // This function is called from outside the controller engine, so we can't
+    // use VERIFY_OR_DEBUG_ASSERT here
```

Comment:
VERIFY_OR_DEBUG_ASSERT can still be used for public functions to verify that other classes are using this correctly. But I'm okay with leaving it as is to avoid more merge conflicts with #1795

Reply:
Actually this function is private, so I don't understand what the problem with VERIFY_OR_DEBUG_ASSERT would be here.

wrapFunctionCode is called from MidiController though, which doesn't check the state of whether the QScriptEngine exists (confusingly, ControllerEngine has a private `m_pEngine` member which is a QScriptEngine and Controller has a private `m_pEngine` member which is a ControllerEngine. In #1795 we renamed the member in ControllerEngine to `m_pScriptEngine` so don't bother changing that in this PR)

Reply:
So there's nothing else to do here?

Reply:
Use VERIFY_OR_DEBUG_ASSERT here, but leave wrapFunctionCode as it is.

Reply:
Done.